### PR TITLE
Weekly VM Updates 2019-10-03

### PR DESCRIPTION
**Pull request by mikloslorinczi**

### CODE REVIEW

File: system_reports/osx-xcode-10.0.x.log [hunk 1]
```diff
@@ -20,7 +20,65 @@
 * Java: openjdk version "1.8.0_222"
 OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
 OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)
-* Flutter: Flutter 1.9.1+hotfix.2 • channel stable • https://github.com/flutter/flutter.git
+Unhandled exception:
```
Comment (koral--) [inline system_reports/osx-xcode-10.0.x.log]: It looks like something went wrong with Flutter on this stack.

Comment (mikloslorinczi) [reply system_reports/osx-xcode-10.0.x.log]: Thanks @koral for spotting this, we will look into it, before rotating our the new stacks updates

**Comment by santiagofm** [comment]
@mikloslorinczi Are these changes live? `osx-vs4mac-stable` seems to have Xcode 11.1 installed...
We tried using `osx-vs4mac-previous-stable` but it still has Xcode 10.1 instead of 10.2.x

**Comment by santiagofm** [comment]
> @mikloslorinczi Are these changes live? `osx-vs4mac-stable` seems to have Xcode 11.1 installed...
> We tried using `osx-vs4mac-previous-stable` but it still has Xcode 10.1 instead of 10.2.x

We ended switching to one of the Xcode stacks

**Comment by mikloslorinczi** [comment]
@santiagofm

We had a minor issue with the system reports last week. I will close this PR and open another one with the correct logs.

osx-vs4mac-stable and osx-vs4mac-edge has Xcode 11.1 and osx-vs4mac-previous-stable has Xcode 10.1

Sorry for the inconvenient.

**Comment by santiagofm** [comment]
> We had a minor issue with the system reports last week. I will close this PR and open another one with the correct logs.

Thank you! No worries, we just were a bit blindsided when about to release our RC that's all. We got it working fine once we figured the Xcode stacks work fine for react-native 😄 

> Now osx-vs4mac-stable, osx-vs4mac-beta and osx-vs4mac-edge has Xcode 11.1 and osx-vs4mac-previous-stable has Xcode 10.1

You mean `osx-vs4mac-previous-stable` has Xcode 10.2.1 though, right?

**Pull request merged**